config/crd/bases/operator.etcd.io_etcdclusters.yaml

```diff
@@ -149,7 +149,8 @@ spec:
                           validityDuration:
                             description: |-
                               ValidityDuration is the expected duration until which the certificate will be valid,
-                              expects in human-readable duration: 100d12h, if empty defaults to 90d
+                              expects in human-readable duration: 100d12h, if empty defaults to 90d for cert-manager
+                              and 365d for auto as per: https://github.com/etcd-io/etcd/blob/main/client/pkg/transport/listener.go#L275
```

Comment:
This link will go out of date quickly. Should we either:
1. Replace with a permalink (commit sha).
2. Replace with a link to some document.
3. Replace with the actual reason why it's like this.